app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,6 @@
+class CustomersController < ApplicationController
+  def show
+    @customer = Customer.find(params[:id])
+    @items = @customer.items
```

Comment:
I would probably just call `@customer.items` in your view and not create an `@items` instance variable in here.